Makefile

```diff
@@ -39,9 +39,35 @@ integ-tests-and-compile: tests
 integ-tests-with-docker: tests-with-docker
 	make compile-with-docker-all
 	make integ-tests
-	
-integ-tests:
+
+prep-python:
 	python3 -m venv .venv
 	.venv/bin/pip install --upgrade pip
 	.venv/bin/pip install requests parameterized
-	.venv/bin/python3 test/integration/local_lambda/test_end_to_end.py
+
+exec-python-e2e-test:
+	TEST_ARCH=${TEST_ARCH} TEST_PORT=${TEST_PORT} .venv/bin/python3 test/integration/local_lambda/test_end_to_end.py
```

Comment:
You should probably confirm this because maybe Makefiles have something special, but I don't think you need the `TEST_ARCH=${TEST_ARCH} TEST_PORT=${TEST_PORT}` (because you would be setting the value of these variables as the same value they already have)

Reply:
Yes I can confirm this env var will be passed along and we don't need to set it every time. Thanks!